infra/terraform/modules/user_nfs_softnas/ec2.tf

```diff
@@ -0,0 +1,142 @@
+data "aws_ami" "softnas" {
+  most_recent = true
```

Comment:
I'm not very familiar with AMI, but this would basically get the last image of SoftNAS right? I wonder if we should stick with a version.